src/lib/hooks.js

```diff
@@ -108,6 +109,10 @@ const api = {
 
 		createOrUpdateGuest(data);
 	},
+  
+	setLanguage(language) {
+		Language.setLanguage(language);
```

Comment:
Please, `export` the `setLanguage` method rather than instantiate a `class` to call the method.